Makefile

```diff
@@ -28,12 +24,20 @@ install-tools:
 
 .PHONY: lint
 lint:
-	golangci-lint run
+	golangci-lint run ./...
+
+.PHONY: up-database
+up-database:
+	docker compose -f test/docker-compose.yml up --quiet-pull -d db --wait
 
 .PHONY: up
 up:
-	docker compose -f test/docker-compose.yml up --quiet-pull -d --wait 
+	docker compose -f test/docker-compose.yml up --wait
 
 .PHONY: down
 down:
-	docker compose -f test/docker-compose.yml down -v
\ No newline at end of file
+	docker compose -f test/docker-compose.yml down -v --remove-orphans
+
+.PHONY: connect
+connect:
+	docker exec -it mysql_db mysql -u root -p'meroxaadmin' meroxadb
```

Comment:
```suggestion
	docker exec -it mysql_db mysql -u root -p 'meroxaadmin' meroxadb
```

Reply:
Don't ask me why, but the space is relevant there, for some mysql specific reason.